helpers/helpers.v1.d/go

```diff
@@ -192,7 +200,8 @@ eval \"\$(goenv init -)\"
 #goenv" > /etc/profile.d/goenv.sh
 
     # Load the environment
-    HOME=$install_dir eval "$(goenv init -)"
+    export HOME=${HOME:-"/root/"}
+    eval "$(goenv init -)"
```

Comment:
I would tend to keep the previous syntax such that `HOME` doesn't get change in the rest of the script which could have side effect idk ... also we probably want to make sure that HOME is the app's install dir and not /root/ ?

Reply:
I think we should remove :
the part :
https://github.com/YunoHost/yunohost/blob/cd12849b94454f06486a7657292d1efb462ef4ee/helpers/helpers.v2.1.d/go#L150-L159
from each helper Go and Ruby; helper 2 and helper2.1

The first part `/etc/profile.d/****` is to set the environment variables at startup of the Bash shell. But we don't need go and ruby environment variables set to each users.

The second part `eval \"\$(goenv init -)\"` is to initialise environment for current user, but we don't need that either, as all we need is done in `_ynh_load_go_in_path_and_other_tweaks` same for ruby